modules/oe_translation_poetry/config/install/field.storage.oe_translation_request.oe_contact_information.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_translation
+    - oe_translation_poetry
+id: oe_translation_request.oe_contact_information
+field_name: oe_contact_information
+entity_type: oe_translation_request
+type: oe_translation_poetry_personal_contact
+settings: {  }
+module: oe_translation_poetry
+locked: false
+cardinality: 1
+translatable: true
```

Comment:
Do we need the field storages translatable? Are we supposed to translate the information from these fields?

Reply:
Indeed they probably shouldn't be, I removed that.